src/lib/graphql.js

```diff
@@ -31,19 +30,39 @@ const SCHEMA_EXCLUDE_LIST_PATTERN =
   /^(?!Query$|Mutation$|Subscription$|__.+$).*$/;
 
 function getDefaultValue(argument) {
+  if (argument.defaultValue === null || argument.defaultValue === undefined) {
+    return undefined;
+  }
+
   if (isListType(argument.type)) {
-    return `[${argument.defaultValue || ""}]`;
+    const defaultValues = Array.isArray(argument.defaultValue)
+      ? argument.defaultValue
+      : [argument.defaultValue];
+
+    return argument.defaultValue !== null && argument.defaultValue !== undefined
+      ? `[${defaultValues
+          .map((defaultValue) => {
+            return printDefaultValue(argument, `"${defaultValue}"`);
+          })
+          .join(", ")}]`
+      : undefined;
```

Comment:
No need for the ternary evaluation, since it is taken care at the beginning of the function.
```suggestion
    return `[${defaultValues
          .map((defaultValue) => {
            return printDefaultValue(argument, `"${defaultValue}"`);
          })
          .join(", ")}]`
```

Reply:
Please move the map operation before the return to make it more readable.

The map returns an array of string ` return printDefaultValue(argument, `"${defaultValue}"`);` what is not correct. The printed array should match the argument type.
- ID, String: ["foo",  "bar"]
- Int: [123, 456]
- Enum: [FOO, BAR]